hack/scaletest/README.md

```diff
@@ -0,0 +1,79 @@
+# Goal
+
+Simulate 1000 clusters in Hive and observe performance. Because we do not have the budget or resources to actually provision 1000 clusters, we will provision one, and adopt 1000 copies of it into Hive. For the purposes of all API communication we will be talking to one cluster. Hive runs 5 threads per controller, so we should not be making excessive concurrent requests to the cluster. However we need to observe for problems from this caveat with regards to SyncSets possibly competing over the same resources and thrashing. If syncing the same resources however, we should always expect the same result, so thrashing should not be possible.
```

Comment:
s/we will provision one, and adopt 1000 copies of it into Hive/we will provision one, and adopt 1000 copies of its clusterdeployment into Hive for testing/s